### PR TITLE
Feature/tl2023 283

### PR DESCRIPTION
**Pull request by Skumarhpa**

### CODE REVIEW

File: src/Sfa.Tl.ResultsAndCertification.Models/Contracts/AdminDashboard/ChangePathwayDetails.cs [hunk 1]
```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Sfa.Tl.ResultsAndCertification.Models.Contracts.AdminDashboard
+{
+    public class ChangePathwayDetails
```
[the remark this answers is not in the document]
Comment (ibai-gonzalez) [reply src/Sfa.Tl.ResultsAndCertification.Models/Contracts/AdminDashboard/ChangePathwayDetails.cs:9]: I see this is only to create the change log JSON. But why these properties and not others? Wouldn't it make more sense to save the created result ID (after the record is saved) and the From and to Results so they get displayed in the UI?

[the remark this answers is not in the document]
Comment (Skumarhpa) [reply src/Sfa.Tl.ResultsAndCertification.Models/Contracts/AdminDashboard/ChangePathwayDetails.cs:9]: We have agreed on saving the from and To fields for the record as JSON details and this been for start year and industry placements also. Using the assessment ID we can track back to result.

[the remark this answers is not in the document]
Comment (Skumarhpa) [reply src/Sfa.Tl.ResultsAndCertification.Models/Contracts/AdminDashboard/ChangePathwayDetails.cs:9]: included resultid

File: src/Sfa.Tl.ResultsAndCertification.Models/Contracts/AdminDashboard/ChangeSpecialismDetails.cs [hunk 2]
```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Sfa.Tl.ResultsAndCertification.Models.Contracts.AdminDashboard
+{
+    public class ChangeSpecialismDetails
```
[the remark this answers is not in the document]
Comment (Skumarhpa) [reply src/Sfa.Tl.ResultsAndCertification.Models/Contracts/AdminDashboard/ChangeSpecialismDetails.cs:9]: inlcuded resultid

**Review by ibai-gonzalez — APPROVED** [review]

**Pull request merged**